Rakefile

```diff
@@ -1,8 +1,12 @@
-require "bundler/gem_tasks"
-require "rspec/core/rake_task"
+require 'bundler/gem_tasks'
+require 'rspec/core/rake_task'
+require 'rubocop/rake_task'
```

Comment:
Hmm i wouldn't agree with the consistency part.  Most project use `'` instead of `"` per option A of the original style guide at https://github.com/bbatsov/ruby-style-guide#consistent-string-literals

Robocop does not have a cop to force all double so there will be no checks on consistency if it switch it back.

Let me know if you still think it should be switched.